pandas/tests/groupby/test_groupby_dropna.py

```diff
@@ -564,9 +564,10 @@ def test_categorical_reducers(reduction_func, observed, sort, as_index, index_ki
         values = expected["y"].values.tolist()
         if index_kind == "single":
             values = [np.nan if e == 4 else e for e in values]
+            expected["y"] = pd.Categorical(values, categories=[1, 2, 3])
```

Comment:
do the test changes indicate bugfixes?

Reply:
Yea - I've added two whatsnew notes, one for this and the other for transform.